web/src/components/HeaderActionItemDropDown.tsx

```diff
@@ -11,19 +13,44 @@ const StyledIcon = styled(Icon)`
   width: 28px;
   height: 28px;
 `
+const ActionBox = styled(Button)<{ $isRTL?: boolean }>`
+  height: 36px;
+  min-width: 122px;
+  display: flex;
+  border-color: black;
+  border-width: 1px;
+  border-style: solid;
+  border-radius: 4px;
+  justify-content: center;
+  align-items: center;
+  flex-direction: ${props => (props.$isRTL ? 'row-reverse' : 'row')};
+`
+
+const StyledText = styled.span`
+  font-weight: bold;
+  font-size: 14px;
+  margin: 0 5px;
+  letter-spacing: 1.25px;
+`
```

Comment:
I could reproduce the issue with two tooltips... This is because this style is also applied to the tooltip text :see_no_evil: If you remove `font-weight` and `letter-spacing`, the issue is gone. Obviously not a good solution though. This might be fixed with the new tooltip introduced in #2847 by @LeandraH. So perhaps it makes sense to wait for that :see_no_evil:

Reply:
So I will put it on hold for now..

Reply:
i think that PR should be merged soonish, hopefully today :) Will let you know

Reply:
It's merged 🥳

Reply:
If you pull from `main`, you should get the new tooltip which lets you use components or text :)

Reply:
Ok thanks..